x-pack/plugins/ml/public/components/job_select_list/job_select_list.html

```diff
@@ -72,7 +73,7 @@
                   value="{{job.id}}"
                   ng-model="$parent.$parent.selectedJobRadio"
                   ng-disabled='job.disabled' />
-                <span ng-class="{'disabled-job': job.disabled}">{{job.id}}</span>
+                <span ng-class="{'disabled-job': job.disabled}" aria-label="Job I.D. {{job.id}}">{{job.id}}</span>
```

Comment:
our general convention for the job ID label is `ID` and not `I.D.`
I don't know how either will be read by a screen reader. but it would probably be best to be consistent with existing uses.

Reply:
Hm, I tried that and it read it phonetically as 'id' and not how we would pronounce i.d. - I thought that might be confusing for listeners. My hunch would be that since it's primarily for accessibility, we'd want it to be easily understood by the user. I'm happy to go with whatever feels best, though.